features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ExploreDatabase.kt

```diff
@@ -83,7 +93,8 @@ abstract class ExploreDatabase : RoomDatabase() {
 
             log.info("Open database {}", config.exploreDatabaseName)
             return dbBuilder
-                .fallbackToDestructiveMigration()
+                //.fallbackToDestructiveMigration()
```

Comment:
I will need to remove the comment.  By removing `.fallbackToDestructiveMigration()` we will get more useful error reporting during development instead of an empty database.

Going forward, using `fallbackToDestructiveMigration()` will be unnecessary.